docs/src/man/getting_started.md

```diff
@@ -427,4 +429,4 @@ df = DataFrame(x = 1, y = 2)
 CSV.write(output, df)
 ```
 
-The behavior of CSV functions can be adapted via keyword arguments. For more information, see `?CSV.read` and `CSV.write`, or checkout the online [CSV.jl documentation](https://juliadata.github.io/CSV.jl/stable/).
\ No newline at end of file
+The behavior of CSV functions can be adapted via keyword arguments. For more information, see `?CSV.read` and `CSV.write`, or checkout the online [CSV.jl documentation](https://juliadata.github.io/CSV.jl/stable/).
```

Comment:
While you're at it, I think I forgot to add `?` before `CSV.write` in my recent PR.